### DISCUSSION TITLE
Get block value on input event

### DISCUSSION THREAD

**Discussion by ivankirshin** [codex-team/editor.js#1913] [Q&A]
Hello!
I want to write a plugin that allows mentioning the user. 
So I need to listen to input events for the current block to parse @ symbol.
How can I do this?
As I understand the inline tool is called only when the user selects a text but not for every input event.
How can I do this? or maybe there is similar plugin?

**Comment by taurenshaman** [comment]
https://github.com/codex-team/editor.js/issues/1163
Maybe we should modify the core code and build ourselves.

I wanna implement some feature like double brackets in Roam Research: [[]], .etc
